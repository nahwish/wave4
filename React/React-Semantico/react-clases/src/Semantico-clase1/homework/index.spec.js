import { render, screen, asFragment } from "@testing-library/react";
import renderer from "react-test-renderer";
import Home from "./index";


describe('',()=>{

   it("Se debe renderizar correctamente", () => {
     const { asFragment } = render(<Home />);
			expect(asFragment()).toMatchSnapshot();
      
   });
})


it("La etiqueta <nav> debe tener el role 'nav' ", () => {
   render(<Home/>)
	const header = screen.getByRole('nav')
   console.log(header)
   expect(header).toBeDefined()
});
