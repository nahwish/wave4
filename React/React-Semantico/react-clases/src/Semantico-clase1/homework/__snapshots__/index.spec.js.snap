// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Se debe renderizar correctamente 1`] = `
<DocumentFragment>
  <nav
    role="nav"
  />
</DocumentFragment>
`;
